### ISSUE TITLE
The solution doesn't build locally if you have multiple dotnet core SDKs installed. (Error	MSB1009	Project file does not exist.)

### ISSUE THREAD

**Issue by stanuku** [solr-express/solr-express#185]
@diegobrum 
I have locally installed .NET Core tools preview for VS 2017. Its SDK version is 1.0.0-preview4-004233. 
I also have tools preview for VS 2015 and SDKs for 1.0.1, 1.0.3 and 1.1.0. 

I cloned the SolrExpress solution locally and I get the build error  in VS2015 ``` Error	MSB1009	Project file does not exist```. If I change global.json as below (added SDK version), then it builds fine. I think the new build system in VS2017 tools looks for .csproj file (not sure yet): 

```
{
    "projects": [ "src", "benchmarks", "test" ],
    "sdk": {
        "version": "1.0.0-preview2-003156" <- this is less than the latest version from VS 2017 tools.
    }
}
```

I believe it is picking up the latest dotnet build tools installed locally. So, we need to add SDK version in the to guide the build process to the correct SDK. Also, I had to manually do 'dotnet restore' on the solution folder level from command prompt, before building it from VS.

Any thoughts?

**Comment by diegobrum** [comment]
For now I only use VS 2015. And VS 2017 is in RC.

I will wait for VS 2015 RTM to change global.json (or other necessary file) :)

**Comment by stanuku** [comment]
@diegobrum: Sounds good.

**Comment by diegobrum** [comment]
Hi @stanuku 
Try again using code implemented in #209
